From 4d36e6fe5ad4233bcda9e0229bba03b25de4bc8b Mon Sep 17 00:00:00 2001
From: Guillaume Papin <guillaume.papin@epitech.eu>
Date: Sat, 3 Nov 2018 11:39:39 +0100
Subject: [PATCH 2/2] schemagen: fully qualify include to
 graphqlservice/GraphQLService.h

Include files are located in $includedir/graphservice,
so it make sense to #include <graphqlservice/GraphQLService.h>
instead of #include "GraphQLService.h".
---
 SchemaGenerator.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SchemaGenerator.cpp b/SchemaGenerator.cpp
index 0b8e17b..42f5867 100644
--- a/SchemaGenerator.cpp
+++ b/SchemaGenerator.cpp
@@ -1179,7 +1179,7 @@ bool Generator::outputHeader() const noexcept
 #include <string>
 #include <vector>
 
-#include "GraphQLService.h"
+#include <graphqlservice/GraphQLService.h>
 
 namespace facebook {
 namespace graphql {
-- 
2.10.0

